openj9-8-rhel8.yaml

```diff
@@ -4,7 +4,7 @@
 
 schema_version: 1
 
-from: "ubi8-minimal:8-released"
+from: "registry.access.redhat.com/ubi8/ubi-minimal:latest"
```

Comment:
I think you can drop the `latest` tag. Just using `registry.access.redhat.com/ubi8/ubi-minimal` should be fine.